contrib/scripts/isc_dhpcd_graphite/isc_dhpcd_graphite.py

```diff
@@ -18,11 +18,14 @@
 import sys
 from time import time
 
+from nav.metrics.names import escape_metric_name
```

Comment:
While this nicely outsources the escaping, this changes the nature of the contrib script in a drastic manner:

Whereas before it was independent of 3rd party libraries and could be deployed on any old DHCP server, it now is dependent on a fully installed NAV library.

You might want to (*gasp*) copy/bundle the escape function with the script :-P

Reply:
Done